packages/sign_in_with_apple/lib/src/widgets/sign_in_with_apple_button.dart

```diff
@@ -20,79 +23,157 @@ class SignInWithAppleButton extends StatelessWidget {
         assert(iconAlignment != null),
         super(key: key);
 
+  /// The callback which will be called when the button is pressed.
   final VoidCallback onPressed;
 
+  /// The text which should be rendered inside the button.
+  ///
+  /// Defaults to `Sign in with Apple`.
   final String text;
 
+  /// The height of the button.
+  ///
+  /// This defaults to `44` according to the Apple Guidelines.
   final double height;
 
+  /// The style of the button according to Apple Guidelines.
+  ///
+  /// This defaults to [SignInWithAppleButtonStyle.black].
   final SignInWithAppleButtonStyle style;
 
+  /// The border radius of the button.
+  ///
+  /// Defaults to `8` pixels.
   final BorderRadius borderRadius;
 
+  /// Controls how the icon should be aligned inside the button.
+  ///
+  /// Defaults to [IconAlignment.center]
```

Comment:
```suggestion
  /// This defaults to [IconAlignment.center]
```

Reply:
I think like the change in https://github.com/aboutyou/dart_packages/pull/55#discussion_r412540251, this would need the `This`. Or is that a style you saw on some Flutter docs / guidlines?